client/src/app/project/project.component.ts

```diff
@@ -37,6 +38,9 @@ export class ProjectComponent implements OnInit {
 
   ngOnInit() {
     this.initProject();
+    this.ingestService.getUserAccount().subscribe(account => {
+      this.userIsWrangler = account.roles.includes('WRANGLER');
```

Comment:
Might be better if this were in the `Account` interface as `isWrangler()` method, so we'd only need to call `account.isWrangler()` to check. However, this probably means having to define an `Account` class instead of just an interface.